java/impl/aws/src/main/java/co/worklytics/psoxy/S3Handler.java

```diff
@@ -68,8 +68,13 @@ StorageEventResponse process(String importBucket, String sourceKey, StorageHandl
             return null;
         }
 
-        byte[] processedData = null;
-        try (ByteArrayOutputStream outputStream = new ByteArrayOutputStream()) {
+        // AWS lambdas have a shared ephemeral storage (shared across invocations) of 512MB
+        // This can be upped to 10GB, but this should be enough as long as we're not processing
+        // lots of large files in parallel.
+        // https://aws.amazon.com/blogs/aws/aws-lambda-now-supports-up-to-10-gb-ephemeral-storage/
```

Comment:
per lambda? or per all lambdas in account?

if per lambda, should never be an issue